pygmt/datasets/tile_map.py

```diff
@@ -102,7 +102,7 @@ def load_tile_map(region, zoom="auto", source=None, lonlat=True, wait=0, max_ret
     Frozen({'band': 3, 'y': 1024, 'x': 1536})
     >>> raster.coords
     Coordinates:
-      * band     (band) int64 0 1 2
+      * band     (band) uint8 0 1 2
       * y        (y) float64 1.663e+05 1.663e+05 1.663e+05 ... 1.272e+05 ...
       * x        (x) float64 1.153e+07 1.153e+07 1.153e+07 ... 1.158e+07 ...
```

Comment:
From https://github.com/GenericMappingTools/pygmt/pull/2125#issuecomment-1454181058, I remembered why I didn't print out the full y and x coordinates here. It's because that would make the docstring too long (>79 characters), so I added `...`  which is supported by https://docs.python.org/3/library/doctest.html#doctest.ELLIPSIS.
```suggestion
      * y        (y) float64 1.663e+05 1.663e+05 1.663e+05 ... 1.272e+05 1.272e+05
      * x        (x) float64 1.153e+07 1.153e+07 1.153e+07 ... 1.158e+07 1.158e+07
```

Reply:
> Perhaps you need to add the `# doctest: +ELLIPSIS` directive after line 103.

I thought so too, but using `...` by itself seems to work? Like, the tests still all pass. I think it was just the int32/int64 diff that was the problem, and pytest still reports the full diff.

Anyways, I've changed the bounding box region in ea9e7dceef00ef1b2c68630a8d59190a33c1037a and the y/x coordinates fit under 79 characters now, so doesn't really matter.